languages/messages/wikia/crowdin.conf

```diff
@@ -0,0 +1,4 @@
+[crowdin.project]
+projectid = "CoreMessagesWikia"
+format = "mediawiki"
+source.file = "MessagesEn.php"
```

Comment:
Looks OK for upload, but the i18n commandline tool doesn't support downloading the translations into the separate language files for MediaWiki right now, so downloading will need some manual work to move them into the right files.